src/acquisition/covidcast/covidcast_meta_cache_updater.py

```diff
@@ -44,13 +52,22 @@ def main(args, epidata_impl=Epidata, database_impl=Database):
 
   # update the cache
   try:
+    metadata_update_start_time = time.time()
     database.update_covidcast_meta_cache(metadata)
+    metadata_update_interval_in_seconds = time.time() - metadata_update_start_time
```

Comment:
this should always be very quick, but it doesnt hurt to measure it anyway

Reply:
acknowledged